src/main/drivers/pwm_output.c

```diff
@@ -215,31 +196,31 @@ void pwmBrushlessMotorConfig(const timerHardware_t *timerHardware, uint8_t motor
     motors[motorIndex]->pwmWritePtr = pwmWriteStandard;
 }
 
-void pwmOneshotPwmRateMotorConfig(const timerHardware_t *timerHardware, uint8_t motorIndex, uint16_t motorPwmRate, uint16_t idlePulse, uint8_t useOneshot42)
+void pwmOneshotPwmRateMotorConfig(const timerHardware_t *timerHardware, uint8_t motorIndex, uint16_t motorPwmRate, bool useOneshot42)
```

Comment:
I'm nitpicking now, but I don't like useOneshot42 being a boolean.... but then again the chance of another OneShot pulse width coming out at this point is slim to none.